src/transformers/models/auto/feature_extraction_auto.py

```diff
@@ -44,6 +44,7 @@
         ("layoutlmv2", "LayoutLMv2FeatureExtractor"),
         ("clip", "CLIPFeatureExtractor"),
         ("perceiver", "PerceiverFeatureExtractor"),
+        ("vit_mae", "ViTFeatureExtractor"),
```

Comment:
I've removed the `feature_extractor_type` from the `preprocessor_config.json` of the checkpoints on the hub.

I guess this is the right way to do it?

Reply:
Yes, this is the right way to provide a default for all models of the type `vit_mae`.